openpype/hosts/traypublisher/plugins/publish/collect_explicit_colorspace.py

```diff
@@ -27,10 +33,38 @@ def process(self, instance):
                 colorspace=colorspace
             )
 
+    @classmethod
+    def apply_settings(cls, project_settings):
+        host = registered_host()
+        host_name = host.name
+        project_name = host.get_current_project_name()
+        config_data = colorspace.get_imageio_config(
+            project_name, host_name,
+            project_settings=project_settings
+        )
+
+        if config_data:
+            filepath = config_data["path"]
+            config_items = colorspace.get_ocio_config_colorspaces(filepath)
+            cls.colorspace_items.extend((
+                (name, name)
+                for name, family in config_items.items()
+            ))
+        else:
+            cls.colorspace_items.extend([
+                ("sRGB", "sRGB"),
+                ("rec709", "rec709"),
+                ("ACES", "ACES"),
+                ("ACEScg", "ACEScg")
+            ])
```

Comment:
Are we sure this makes sense? Isn't it weird to default to some random entries like this? I know these are popular entries, but they might not exist at all in certain ocio configs. Would it make more sense to maybe just disable the EnumDef entry completely (like hide it?) and log a debug message if no OCIO config is set with an explanation as to why the feature is disabled?

Reply:
good point, make sense to just hide it.